src/D2L.Bmx/UpdateHandler.cs

```diff
@@ -87,6 +87,13 @@ internal class UpdateHandler( IGitHubClient github, IVersionProvider versionProv
 		}
 
 		Console.WriteLine( $"BMX updated to v{latestVersion} successfully!" );
+
+		try {
+			Directory.Delete( workspaceDir, recursive: true );
+		} catch {
+			// Ignore any errors from clean-up.
+			// This just isn't always possible (e.g. when on Windows), and it's harmless anyway.
+		}
```

Comment:
this will delete as much stuff as it can. on Windows it should be everything except the old executable. on Mac & Linux it should be everything.